website/src/controllers/api-v3/user.js

```diff
@@ -41,6 +41,69 @@ api.getUser = {
   },
 };
 
+function _cleanChecklist (task) {
+  let checklistIndex = 0;
+  _.forEach(task.checklist, (c) => {
```

Comment:
when you use _.forEach the index is passed as the second argument, so you could do 

```
_.forEach(task.checklist, (c, i) => {
     c.text = `item i`;
});
```